docker-compose.yml

```diff
@@ -43,6 +46,5 @@ services:
       - elasticsearch
 
 networks:
-
   elk:
-    driver: bridge
+    driver: overlay
```

Comment:
Forcing the driver type to `overlay` will make this stack compatible with Swarm only.